kernel/runtime/kobject/runtime/ExecutionContext.hh

```diff
@@ -94,13 +106,11 @@ namespace mythos {
         }
 
     protected:
-        static void start(StartFun main, void* userctx, CapPtr ec) { 
-            localEC = ec;
+        static void start(StartFun main, void* userctx, uintptr_t) { 
```

Comment:
shouldn't we just abandon the unused parameter here?
If the thread is not created through the musl libc, then the user is responsible to pass this information via the datastructre where userctx points to.

Reply:
I agree, I will remove the parameter completely.